src/cli/index.js

```diff
@@ -142,7 +142,7 @@ if (outputWrapper) {
 if (command.noArguments && commander.args.length) {
   reporter.error(reporter.lang('noArguments'));
   reporter.info(command.getDocsInfo);
-  process.exit(1);
+  process.exitCode = 1;
```

Comment:
Not shortcutting the whole point with this change. We should not need the shortcut behavior and turns out we really don't since all tests are passing. It was only necessary at one point so I left it that way.